tox.ini

```diff
@@ -23,7 +23,11 @@ envlist =
     # asyncio without Tornado.
     asyncio-{pypy35,pypy36,py35,py36,py37,py38,py39},
 
+<<<<<<< HEAD
```

Comment:
Looks like you didn't merge this file correctly - can you fix it? It should just read:
```
# Test PyMongo 3.12 because 4.x breaks some APIs
py3-pymongo-v3.12,
```

Reply:
removed this already